docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md

```diff
@@ -0,0 +1,772 @@
+   [runcts] OUT => [javatest.batch] Completed running 175 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 175
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+
+Stage Name: ejb30/lite/lookup
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+   [runcts] OUT => [javatest.batch] Completed running 30 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 30
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+
+Stage Name: ejb30/lite/naming
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+   [runcts] OUT => [javatest.batch] Completed running 54 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 54
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+
+Stage Name: ejb30/lite/nointerface
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+   [runcts] OUT => [javatest.batch] Completed running 60 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 60
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+
+Stage Name: ejb30/lite/packaging
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+   [runcts] OUT => [javatest.batch] Completed running 211 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 211
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+
+Stage Name: ejb30/lite/singleton
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+   [runcts] OUT => [javatest.batch] Completed running 230 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 230
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 10
```

Comment:
@arjantijms Here is a little strange. Is this a TCK error?

Reply:
Thanks for checking! Probably a copy paste error in some way by me.

I copied everything from https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/master/1898/artifact/ you have to download each of the 150MB files, then hunt in the logs for the right snippet. It's very error prone and easy to make mistakes :(